tests/e2e/lib.sh

```diff
@@ -637,6 +656,18 @@ remove_existing_stackrox_resources() {
     ) 2>&1 | sed -e 's/^/out: /' || true
 }
 
+remove_compliance_operator_resources() {
```

Comment:
Not super useful for CI, but useful for local development using `run-compliance-e2e.sh` directly.